### DISCUSSION TITLE
Document downgrage process

### DISCUSSION THREAD

**Discussion by zamazan4ik** [seaweedfs/seaweedfs#3409] [Ideas]
Hi.

After reading the documentation it is not clear for me, how the downgrade process should be performed over the solution. Could you please describe somewhere in the documentation:

- How to upgrade seeweedfs to the older version (hopefully with no downtime). I have read https://github.com/seaweedfs/seaweedfs/wiki/FAQ#how-to-upgrade-from-release-1xx-to-1yy but it has no mention about zero-downtime upgrades
- How to downgrade seeweedfs to the older version (hopefully with no downtime)
- Maybe some compatibility policies and notes between versions

Thanks in advance!

**Comment by chrislusf** [comment]
downgrade is generally not supported unless minor version changes. In general, it would be just change the binary and restart the instance.
